src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamNetworkStreamTest.cs

```diff
@@ -171,7 +171,7 @@ public async Task SslStream_NetworkStream_Renegotiation_Succeeds(bool useSync)
             }
         }
 
-        [Theory]
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindows7))]
```

Comment:
?

Reply:
Yah, the test failed on Windows7 and I did not have chance to investigate. I'm not sure how important that is but we can possibly throw PNSP if we cannot figure out how to make it work. The flow will need to probably change once we incorporate Linux so I did not want to invest too much into it at the moment.  

Main goal of this PR was to provide something useable so you can experiment with Kestrel and provide feedback @Tratcher

Reply:
Thanks, Win7 support isn't critical, so long as we get a good explanation for why it doesn't work. Http.Sys had an issue on Win7 that might be related: https://github.com/dotnet/aspnetcore/blob/8f564897f68944d5f9bf9bded45160fcf3fb0329/src/Servers/HttpSys/src/HttpSysListener.cs#L21-L29

Reply:
Thanks for the link. That is good to know.